src/main/webapp/script.js

```diff
@@ -558,6 +593,62 @@ function backToHomepage() {
   eraseQueryStringFromUrl();
 }
 
+/** 
+ * Logic to change the follower status of the user regarding a
+ * specific question. 
+ * 
+ * @param {boolean} userFollowsQuestion
+ * @param {int} questionId
+ */
+function followUnfollow(userFollowsQuestion, questionId) {
```

Comment:
I think the method name might be more readable if it was something like "updateFollowingStatus" or something along those lines.

Reply:
done